onnx/test/shape_inference_test.py

```diff
@@ -8770,6 +8778,39 @@ def test_one_hot_encoder(self) -> None:
             ],
         )
 
+    @unittest.skipUnless(ONNX_ML, "ONNX_ML required to test ai.onnx.ml operators")
+    def test_zip_map(self) -> None:
```

Comment:
Should we add a test to check the inference is correct when classlabels_strings is set up?

Reply:
I can add it here for completeness of `test_zip_map` at the current version, though the primary goal of the PR is to have map-type inference work at all so I don't think I'll add other test cases.